doc/source/index.rst

```diff
@@ -1,51 +1,47 @@
-scale due to differences in incident flux, for example. ``PDFmorph`` will 
-do its best to correct for these benign effects before computing and 
-plotting the difference curve. One measured PDF (typically that 
-collected under higher temperature) is identified as the target PDF and 
-the second PDF is then morphed by "stretching" (changing the r-axis to
-simulate a uniform lattice expansion), "smearing" (broadening peaks 
-through a uniform convolution to simulate increased thermal motion), and 
-"scaling" (self-explanatory). PDFmorph will vary the amplitude of the 
-morphing transformations to obtain the best fit between the morphed and 
-the target PDFs, then plot them on top of each other with the difference 
-plotted below.
-
-There are also a few other morphing transformations in the program.
-If no morphing transformation is specified, ``PDFmorph`` will return just
-the plotted PDFs.
-
-Finally, we note that though ``PDFmorph`` should work on other spectra 
-that are not PDFs, it has not been extensively tested beyond the PDF.
-
+(PDFs) or x-ray diffraction patterns (XRD). It was designed to help a
+researcher answer the question: "What are the structural signals composing
+my chemical reaction and how do they change with time?"
+
+One approach is to take the conventional nonnegative matrix factorization
+model and extend it by introducing a stretching factor matrix that accounts
+for isotropic stretching of measured signals and returns components that explain
+variability beyond this stretching. Conventional nonnegative matrix factorization
+(nmf) obtains component signals and weightings of the component signals from a
+series of PDF or XRD files. However, this model assumes the components themselves
+are constant in time, making this model unable to capture the behavior of
+
+``snmf`` will extend the behavior of nmf by introducing a stretching factor
+at each moment in time that isotropically stretches the component signals.
+The algorithm will attempt to find three matrices, a "component" matrix that
+stores the component signals, a "stretching factor" matrix that stores the
+stretching factors of the components at each moment in time, and a "weights"
+matrix that stores the weights of each component at each moment in time.
+``snmf`` will then plot the components, stretching factors, and weights.
+
+It is important to note that the user must specify the number of component
+signals to obtain from the experimental data. Non-physical results may be
+obtained if the number of anticipated component signals is too high.
 
 Authors
 -------
 
-``PDFmorph`` is developed by members of the Billinge Group at 
+``snmf`` is developed by members of the Billinge Group at
 Columbia University and Brookhaven National Laboratory including 
-Christopher L. Farrow, Christopher J. Wright, Pavol Juhás, Chia-Hao
-(Timothy) Liu, S. Matthew Román, and Simon J. L. Billinge.
+Ran Gu, Adeolu Ajayi, Qiang Du, and Simon J. L. Billinge.
```

Comment:
It may be better to separate who developed the algo and who contributed to the development of the code.

Reply:
I will do this